### PR TITLE
Fix error collides with imported package name

### PR DESCRIPTION
**Pull request by Moriadry-zz**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #
Fix error collides with imported package name for e2e test

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
NONE
```

### CODE REVIEW

File: test/e2e/framework/kubelet_stats.go [hunk 1]
```diff
@@ -494,10 +494,10 @@ func GetKubeletHeapStats(c clientset.Interface, nodeName string) (string, error)
 	if errRaw != nil {
 		return "", err
 	}
-	kubeletstatsv1alpha1 := string(raw)
+	kubeletstatsv1alpha1str := string(raw)
```
Comment (BenTheElder) [inline test/e2e/framework/kubelet_stats.go]: what about just `kubeletStats` ? or `heapStats`? it's unusual to put the type in a variable name in go and this is pretty long. 

https://github.com/golang/go/wiki/CodeReviewComments#variable-names

```suggestion
	heapStats := string(raw)
```

File: test/e2e/framework/kubelet_stats.go [hunk 2]
```diff
@@ -602,15 +602,15 @@ func (r *resourceCollector) collectStats(oldStatsMap map[string]*kubeletstatsv1a
 func (r *resourceCollector) GetLatest() (e2ekubelet.ResourceUsagePerContainer, error) {
 	r.lock.RLock()
 	defer r.lock.RUnlock()
-	kubeletstatsv1alpha1 := make(e2ekubelet.ResourceUsagePerContainer)
+	kubeletstatsv1alpha1map := make(e2ekubelet.ResourceUsagePerContainer)
```
Comment (BenTheElder) [inline test/e2e/framework/kubelet_stats.go]: `usagePerContainer`?
```suggestion
	usagePerContainer := make(e2ekubelet.ResourceUsagePerContainer)
```

File: test/e2e/framework/framework.go [hunk 3]
```diff
@@ -569,7 +569,7 @@ func (f *Framework) CreateServiceForSimpleApp(contPort, svcPort int, appName str
 // CreatePodsPerNodeForSimpleApp creates pods w/ labels.  Useful for tests which make a bunch of pods w/o any networking.
 func (f *Framework) CreatePodsPerNodeForSimpleApp(appName string, podSpec func(n v1.Node) v1.PodSpec, maxCount int) map[string]string {
 	nodes := GetReadySchedulableNodesOrDie(f.ClientSet)
-	labels := map[string]string{
+	labelmaps := map[string]string{
```
Comment (BenTheElder) [inline test/e2e/framework/framework.go]: nit: It's only one map, not "maps", `podLabels` would be more descriptive
```suggestion
	podLabels := map[string]string{
```

File: test/e2e/framework/lifecycle/upgrade.go [hunk 4]
```diff
@@ -43,12 +43,12 @@ func RealVersion(s string) (string, error) {
 }
 
 func traceRouteToMaster() {
-	path, err := exec.LookPath("traceroute")
+	traceroutepath, err := exec.LookPath("traceroute")
```
Comment (BenTheElder) [inline test/e2e/framework/lifecycle/upgrade.go]: for the `exec.Command` below which appears to be the only usage, `exec.Command(traceroute, ...)` is very readable but a bit less verbose

```suggestion
	traceroute, err := exec.LookPath("traceroute")
```

File: test/e2e/framework/providers/gce/gce.go [hunk 5]
```diff
@@ -170,8 +170,8 @@ func (p *Provider) EnsureLoadBalancerResourcesDeleted(ip, portRange string) erro
 	}
 
 	return wait.Poll(10*time.Second, 5*time.Minute, func() (bool, error) {
-		e2eservice := p.gceCloud.ComputeServices().GA
-		list, err := e2eservice.ForwardingRules.List(project, region).Do()
+		e2ecomputeservice := p.gceCloud.ComputeServices().GA
```
Comment (BenTheElder) [inline test/e2e/framework/providers/gce/gce.go]:
```suggestion
		computeService := p.gceCloud.ComputeServices().GA
```

File: test/e2e/framework/test_context.go [hunk 6]
```diff
@@ -374,7 +374,7 @@ func createKubeConfig(clientCfg *restclient.Config) *clientcmdapi.Config {
 	userNick := "user"
 	contextNick := "context"
 
-	config := clientcmdapi.NewConfig()
+	configcmd := clientcmdapi.NewConfig()
```
Comment (BenTheElder) [inline test/e2e/framework/test_context.go]:
```suggestion
	configCmd := clientcmdapi.NewConfig()
```

File: test/e2e/framework/util.go [hunk 7]
```diff
@@ -3053,14 +3053,14 @@ func getMasterAddresses(c clientset.Interface) (string, string, string) {
 	internalIP = eps.Subsets[0].Addresses[0].IP
 
 	// Populate the external IP/hostname.
-	url, err := url.Parse(TestContext.Host)
+	hosturl, err := url.Parse(TestContext.Host)
```
Comment (BenTheElder) [inline test/e2e/framework/util.go]:
```suggestion
	hostURL, err := url.Parse(TestContext.Host)
```

File: test/e2e/framework/util.go [hunk 8]
```diff
@@ -3241,7 +3241,7 @@ func DumpDebugInfo(c clientset.Interface, ns string) {
 
 // DsFromManifest reads a .json/yaml file and returns the daemonset in it.
 func DsFromManifest(url string) (*appsv1.DaemonSet, error) {
-	var controller appsv1.DaemonSet
+	var dscontroller appsv1.DaemonSet
```
Comment (BenTheElder) [inline test/e2e/framework/util.go]: this is a daemonset, not a controller?
```suggestion
	var ds appsv1.DaemonSet
```

File: test/e2e/framework/viperconfig/viperconfig_test.go [hunk 9]
```diff
@@ -44,7 +44,7 @@ func TestViperConfig(t *testing.T) {
 		config.AddOptionsToSet(flags, &context, "")
 	})
 
-	config := `
+	viperconfig := `
```
Comment (BenTheElder) [inline test/e2e/framework/viperconfig/viperconfig_test.go]:
```suggestion
	viperConfig := `
```

File: test/e2e/framework/util.go [hunk 10]
```diff
@@ -353,17 +353,17 @@ func SkipUnlessTaintBasedEvictionsEnabled() {
 
 // SkipIfContainerRuntimeIs skips if the container runtime is included in the runtimes.
 func SkipIfContainerRuntimeIs(runtimes ...string) {
-	for _, runtime := range runtimes {
-		if runtime == TestContext.ContainerRuntime {
-			skipInternalf(1, "Not supported under container runtime %s", runtime)
+	for _, containerruntime := range runtimes {
```
Comment (BenTheElder) [inline test/e2e/framework/util.go]:
```suggestion
	for _, containerRuntime := range runtimes {
```

or just
```suggestion
	for _, rt := range runtimes {
```

go encourages short names for local variables with limited scope

File: test/e2e/framework/util.go [hunk 11]
```diff
@@ -2105,7 +2105,7 @@ func AddOrUpdateAvoidPodOnNode(c clientset.Interface, nodeName string, avoidPods
 			if !apierrs.IsConflict(err) {
 				ExpectNoError(err)
 			} else {
-				e2elog.Logf("Conflict when trying to add/update avoidPods %v to %v with error %v", avoidPods, nodeName, err)
+				e2elog.Logf("Conflict when trying to add/update avoidPonds %v to %v", avoidPods, nodeName)
```
Comment (BenTheElder) [inline test/e2e/framework/util.go]: why was this changed? ponds? please revert this line

File: test/e2e/framework/util.go [hunk 12]
```diff
@@ -3268,16 +3268,16 @@ func DsFromManifest(url string) (*appsv1.DaemonSet, error) {
 		return nil, fmt.Errorf("Failed to read html response body: %v", err)
 	}
 
-	json, err := utilyaml.ToJSON(data)
+	jsondata, err := utilyaml.ToJSON(data)
```
Comment (BenTheElder) [inline test/e2e/framework/util.go]:
```suggestion
	dataJSON, err := utilyaml.ToJSON(data)
```

**Review by BenTheElder — CHANGES_REQUESTED** [review]
Hi, thanks for the PR, I see at least one change that is bad though and am suggestion a number of possibly more idiomatic / informative names 😅

**Comment by Moriadry** [comment]
Thanks for your review. I'm not familiar with variable name style in go, and  will fix it ASAP.

**Review by BenTheElder — APPROVED** [review]
/approve
needs a rebase, sorry about that, and thanks for the PR!

**Comment by Moriadry** [comment]
@BenTheElder 
Hi, I used wrong command `git merge` to resolve it, It's okay then.
So, will it be done if I squash my commits?

**Comment by BenTheElder** [comment]
the current git state looks fine I think

**Pull request merged**